apps/crn-server/test/mocks/user-data-provider.mock.ts

```diff
@@ -1,8 +1,8 @@
-import { UserDataProvider } from '../../src/data-providers/users.data-provider';
+import { UserDataProvider } from '../../src/data-providers/types';
 
-export const userDataProviderMock: jest.Mocked<UserDataProvider> = {
+export const userDataProviderMock = {
   fetchById: jest.fn(),
```

Comment:
since its always these for methods now maybe we want a helper function to generate these? 
Although since create and update may be no-ops it's worth addressing that in that helper method too

Reply:
So if i understand correctly we would had  one `getDataProviderMock` function instead of these specific mock for each content type.